### PR TITLE
[ANDROID] SplashScreen added

### PR DESCRIPTION
**Pull request by RCH010**
this is the one...
The Splash Screen was created with a 1.5 seconds duration.
It is necessary to change the image of the beginning for the logo, and to verify the hexadecimal of the color of background (@color/colorBackgroundSplashScreen)

AndroidManifest.xml, colors.xml, SplashScreen.kt, y activity_splash_screen.xml

### CODE REVIEW

File: android/app/src/main/java/tech/voidmx/chupapp/SplashScreen.kt [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        setContentView(R.layout.activity_splash_screen)
+
+        val background = object : Thread() {
+            override fun run() {
+                try {
+                    Thread.sleep(1500)
+
+                    val intent = Intent(baseContext, MainActivity::class.java)
+                    startActivity(intent)
+                }catch (e : Exception) {
```
Comment (inakijaneiro) [inline android/app/src/main/java/tech/voidmx/chupapp/SplashScreen.kt]: Te falto un espacio despues del bracket

File: android/app/src/main/java/tech/voidmx/chupapp/SplashScreen.kt [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import android.os.Bundle
+import java.lang.Exception
+
+class SplashScreen : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_splash_screen)
+
+        val background = object : Thread() {
```
Comment (sgdiosdado) [inline android/app/src/main/java/tech/voidmx/chupapp/SplashScreen.kt]: El nombre de esta constante no es self-explanatory. No entiendo que se supone que almacena. 🤔

File: android/app/src/main/java/tech/voidmx/chupapp/SplashScreen.kt (hunk 1 above)
Comment (sgdiosdado) [inline android/app/src/main/java/tech/voidmx/chupapp/SplashScreen.kt]: Agrega un espacio entre `}` y `catch`. 🤓

**Review by sgdiosdado — DISMISSED** [review]

**Review by inakijaneiro — DISMISSED** [review]
Great!

**Review by inakijaneiro — DISMISSED** [review]

**Review by sgdiosdado — APPROVED** [review]

**Pull request merged**